arches/app/media/js/views/components/search/search-results.js

```diff
@@ -56,18 +56,21 @@ define([
                 this.restoreState();
 
                 this.mapFilter = this.getFilterByType("map-filter-type", false);
-                this.mapFilter.subscribe(mapFilter => {
-                    if (mapFilter) {
-                        this.mapFilter = mapFilter;
-                    }
-                }, this);
-                this.selectedTab.subscribe(function (tab) {
-                    if (tab === "map-filter-type") {
-                        if (ko.unwrap(this.mapFilter.map)) {
-                            this.mapFilter.map().resize();
+                //Check if exist the map filter
+                if (this.mapFilter !== null){
```

Comment:
It would be safer just to check if `this.mapFilter` has any value rather than specifically for `null`.
```suggestion
                if (this.mapFilter){
```